ts/src/okx.ts

```diff
@@ -4855,17 +4808,47 @@ export default class okx extends Exchange {
         //         "type": "0"
         //     }
         //
-        const id = this.safeString (transfer, 'transId');
+        // fetchTransfers
+        //
+        //     {
+        //         "bal": "70.6874353780312913",
+        //         "balChg": "-4.0000000000000000", // negative means "to funding", positive meand "from funding"
+        //         "billId": "588900695232225299",
+        //         "ccy": "USDT",
+        //         "execType": "",
+        //         "fee": "",
+        //         "from": "18",
+        //         "instId": "",
+        //         "instType": "",
+        //         "mgnMode": "",
+        //         "notes": "To Funding Account",
+        //         "ordId": "",
+        //         "pnl": "",
+        //         "posBal": "",
+        //         "posBalChg": "",
+        //         "price": "0",
+        //         "subType": "12",
+        //         "sz": "-4",
+        //         "to": "6",
+        //         "ts": "1686676866989",
+        //         "type": "1"
+        //     }
+        //
+        const id = this.safeString2 (transfer, 'transId', 'billId');
         const currencyId = this.safeString (transfer, 'ccy');
         const code = this.safeCurrencyCode (currencyId, currency);
-        const amount = this.safeNumber (transfer, 'amt');
+        let amount = this.safeNumber (transfer, 'amt');
         const fromAccountId = this.safeString (transfer, 'from');
         const toAccountId = this.safeString (transfer, 'to');
         const accountsById = this.safeValue (this.options, 'accountsById', {});
         const fromAccount = this.safeString (accountsById, fromAccountId);
         const toAccount = this.safeString (accountsById, toAccountId);
-        const timestamp = this.milliseconds ();
+        const timestamp = this.safeInteger (transfer, 'ts', this.milliseconds ());
         const status = this.safeString (transfer, 'state');
```

Comment:
@ttodua we're missing the status parsing here, example from  a different exchange:
```Javascript
    parseTransferStatus (status) {
        const statuses = {
            'prepared': 'pending',
            'confirmed': 'ok',
            'cancelled': 'cancelled',
            'waiting_for_admin': 'pending',
        };
        return this.safeString (statuses, status, status);
    }
```

Reply:
but the status-related info is not provided in the raw response. should we set it to "ok" directly?

Reply:
@ttodua in that case why do we have this line here: ` const status = this.safeString (transfer, 'state');`

Reply:
but that line might be for other methods (i.e. `fetchTransfer` as the commented response sample where `state` key appears, belongs to it, which was there before I added `fetchTransfers`).

Reply:
> but that line might be for other methods
@ttodua exactly, that is an issue because we're using the provided raw value instead of the unified one 😅

Reply:
@ttodua thanks!